.build/tasks/Common.Functions.ps1

```diff
@@ -209,3 +209,345 @@ function Get-CodeCoverageOutputFileEncoding
 
     return $codeCoverageOutputFileEncoding
 }
+
+function Merge-JaCoCoReports
+{
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.Xml.XmlDocument]
+        $OriginalDocument,
```

Comment:
I would probably have accepted Files in this function, and only within the function read & parse the XML. That would have made the calling function a bit simpler.